src/loader.js

```diff
@@ -44,9 +44,10 @@ const MiniCssExtractPlugin = require("./index");
  * @returns {string}
  */
 function hotLoader(content, context) {
-  const accept = context.locals
+  const hasLocals = !!(context.locals && Object.keys(context.locals).length);
+  const accept = hasLocals
     ? ""
-    : "module.hot.accept(undefined, cssReload);";
```

Comment:
Why you remove it?

Reply:
self accept is enough, it will update when dispose: https://webpack.js.org/api/hot-module-replacement/